app/src/main/res/values/strings.xml

```diff
@@ -1079,6 +1079,12 @@ If you are overwhelmed by the number of quests, you can always fine-tune which q
     <string name="quest_presets_selected">Selected</string>
     <string name="quest_presets_delete_message">"Delete preset “%s”? The quest selection and order for it will be irrevocably lost."</string>
     <string name="quest_picnicTableCover_title">Is this picnic table covered (protected from rain)?</string>
+    <string name="quest_bicycle_barrier_type_title">What type of bicycle barrier is this?</string>
+    <string name="quest_barrier_bicycle_type_diagonal">Diagonal</string>
```

Comment:
I agree with @riQQ (see https://github.com/streetcomplete/StreetComplete/issues/3361#issuecomment-961432327) that "Diagonal to the way"/"Diagonal to the path" would be better, as the `tilted` barrier is also "diagonal".

*(commenting here, so the discussion can be resolved)*